app/views/items/_item_type_form.html.erb

```diff
@@ -1,13 +1,44 @@
-<% if @item.book? %>
-<h1><%=I18n.t("items.type.book")%></h1>
- <%= render "book_form", item: @item %>
-<% elsif @item.movie? %>
-<h1><%=I18n.t("items.type.movie")%></h1>
- <%= render "movie_form", item: @item %>
-<% elsif @item.game? %>
-<h1><%=I18n.t("items.type.game")%></h1>
- <%= render "game_form", item: @item %>
-<% else @item.other? %>
-<h1><%=I18n.t("items.type.other")%></h1>
- <%= render "other_form", item: @item %>
+<%= form_with(model: @item) do |form| %>
+  <% if @item.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(@item.errors.count, "error") %> prohibited this item from being saved:</h2>
+      <ul>
+        <% @item.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  
+  <div class="mb-2">
+    <%= form.label :name, I18n.t("items.form.all.name"), class: "form-label" %>
+    <%= form.text_field :name, class: "form-control" %>
+  </div>
+
+  <div class="mb-2">
+    <%= form.label :description, I18n.t("items.form.all.description"), class: "form-label" %>
+    <%= form.text_field :description, class: "form-control" %>
+  </div>
+
+  <div class="mb-2">
+    <%= form.label I18n.t("items.form.all.max_borrowing_days"), class: "form-label" %>
+    <%= form.number_field :max_borrowing_days, min: 0, max: 365, class: "form-control" %>
+  </div>
+
+  <div class="mb-2">
+    <%= form.label I18n.t("items.form.all.max_reservation_days"), class: "form-label" %>
+    <%= form.number_field :max_reservation_days, min: 0, max: 365, class: "form-control"%>
+  </div>
```

Comment:
Wofür benötigt ihr eigentlich die `_form.html.erb` wenn ihr hier die Standard Attribute nochmals auflistet? Könnte man nicht stattdessen einfach `_form.html.erb` rendern oder diese entfernen? Bei mir schlagen dann viele Tests fehl, aber ihr habt sicherlich mehr Ahnung davon wie das mit dem ganzen Rendering funktioniert. Nur in `_form.html.erb` und hier steht jetzt mehr oder weniger dasselbe.

Reply:
das hatte ich mir auch gedacht, dachte mir aber vielleicht steht die noch irgendwo als Artefakt drin. Ich hab sie deleted, Tests laufen noch und es gibt keine offensichtlichen Probleme.